static/app/components/assistant/guideAnchor.tsx

```diff
@@ -237,7 +239,7 @@ type WrapperProps = React.PropsWithChildren<{disabled?: boolean} & Props>;
  * anchors on the page to determine which guides can be shown on the page.
  */
 function GuideAnchor({disabled, children, ...rest}: WrapperProps) {
-  if (disabled || window.localStorage.getItem('hide_anchors') === '1') {
```

Comment:
Removed the localstorage logic since only the sandbox used it and it's not using it anymore

Reply:
Thank you!